data/json/vehicleparts/appliances.json

```
@@ -102,5 +102,46 @@
       { "item": "scrap", "count": [ 1, 2 ] }
     ],
     "flags": [ "CIRCLE_LIGHT", "APPLIANCE", "ENABLED_DRAINS_EPOWER", "CTRL_ELECTRONIC" ]
+  },
+  {
+    "type": "vehicle_part",
+    "id": "ap_solar_panel",
+    "name": { "str": "mounted solar panel" },
+    "symbol": "#",
+    "categories": [ "energy" ],
+    "color": "yellow",
```

```suggestion
    "categories": [ "energy" ],
    "location": "structure",
    "color": "yellow",
```
appliances need a location entry